### DISCUSSION TITLE
可以添加一个自定义倍速快捷键吗？

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#1438] [功能建议 / Ideas]
这个想法是自己最近一直在使用Video Speed Controller，但是在播放列表时，就容易失效。需要刷新页面才能够使用。
这个脚本能不能添加一个自定义倍速快捷键（例如可以使用X或者B键盘都可以），用户自定义一个播放速度，点击快捷键的时候，能够以设定倍速播放，再次点击能够恢复上一次的倍速。

**Comment by LonelySteve** [comment]
你和 #1441 说的有点类似，切换到上次选择的倍速这个功能可以考虑实现，不过目前脚本内置的 VideoSpeedController 只支持原生倍速和 2.5x，3x 这两个扩展倍速，暂时不能由用户指定扩展倍速（考虑到扩展原生倍速菜单的技术原因，即使实现可能也只能向 2.0x 以上的倍速自定义扩展）

**Reply by kethoven** [reply]
> 
> 
> 你和 #1441 说的有点类似，切换到上次选择的倍速这个功能可以考虑实现，不过目前脚本内置的 VideoSpeedController 只支持原生倍速和 2.5x，3x 这两个扩展倍速，暂时不能由用户指定扩展倍速（考虑到扩展原生倍速菜单的技术原因，即使实现可能也只能向 2.0x 以上的倍速自定义扩展）

好的！

**Comment by Wanglizheng1** [comment]
#1230

**Comment by Geeyun-JY3** [comment]
现在还是有用户在提这个需求：#4443、#4446，还做吗？
V2 目前的`记忆倍速`组件配合`快捷键扩展`中的`切换倍速`快捷键也并不能完全满足需求，毕竟破站官方也早就搞了两个键位莫名其妙的直接切换至指定倍速的快捷键：https://github.com/the1812/Bilibili-Evolved/discussions/2231#discussioncomment-1665055
- <kbd>Shift</kbd> + <kbd>1</kbd>：一倍速（正常倍速）
- <kbd>Shift</kbd> + <kbd>2</kbd>：二倍速
